.github/workflows/CI.yml

```
@@ -24,6 +28,82 @@ jobs:
         name: wheels
         path: dist
 
+  musllinux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - x86_64-unknown-linux-musl
+          - i686-unknown-linux-musl
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+          architecture: x64
+      - name: Build wheels
+        uses: messense/maturin-action@v1
+        with:
+          target: ${{ matrix.target }}
+          manylinux: musllinux_1_2
+          args: --release --out dist --find-interpreter
```

```suggestion
          args: --release --out dist -i 3.7 3.8 3.9 3.10 3.11 3.12
```

seems like a version issue, it only built for 3.10 before.